x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/get_is_alert_suppression_active.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { LicensingPluginSetup } from '@kbn/licensing-plugin/server';
+
+import { firstValueFrom } from 'rxjs';
+
+import type { AlertSuppressionCamel } from '../../../../../common/api/detection_engine/model/rule_schema';
+
+interface GetIsAlertSuppressionActiveParams {
+  alertSuppression: AlertSuppressionCamel | undefined;
+  isFeatureDisabled: boolean | undefined;
+  licensing: LicensingPluginSetup;
+}
+
+/**
+ * checks if alert suppression is active:
+ * - rule should have alert suppression config
+ * - feature flag should not be disabled
+ * - license should be platinum
+ */
+export const getIsAlertSuppressionActive = async ({
```

Comment:
Note: this exists as [`isAlertSuppressionActive`](https://github.com/elastic/kibana/pull/176422/files#diff-f3b55ecbc2dfc70ebdb1af1ac6591b982d63259d9ae1271320ad4eca7bf8647aR20) on the EQL suppression branch. We should probably reconcile those once this is merged.